src/routes/changelog/(entries)/2024-04-25.markdoc

```diff
@@ -0,0 +1,12 @@
+---
+layout: changelog
+title: Announcing Appwrite's Startups program
+date: 2024-04-25
+cover: /images/changelog/2024-04-25.png
```

Comment:
why do we add same image twice? Why not just use /images/blog/startups.png ?

Reply:
We do the same for all the other PRs, like Hippa compliant and stuff, probably to keep no dependency between one other?